### PR TITLE
FSharpLint: upgrade to latest version 0.21.3

### PR DESCRIPTION
**Pull request by parhamsaremi**
Enable FavourStaticEmptyFields rule.

### CODE REVIEW

**Comment by knocte** [comment]
If this was enough, I would have done it myself. You need to increase the config file to include new rules introduced in the new version.

**Comment by parhamsaremi** [comment]
> If this was enough, I would have done it myself. You need to increase the config file to include new rules introduced in the new version.

hmmm sorry. I was shocked with how easy it was and wanted to ask you if I did it correctly in the EOD message but forgot about it when I was talking to Afshin about the bug. I'll do this.

File: NOnion/Utility/Base32Util.fs [hunk 1]
```diff
@@ -163,4 +167,4 @@ module Base32Util =
                         tail
                 | _ -> result
 
-            [ for c in text -> c ] |> parse [] |> List.toArray
+            [ for c in text -> c ] |> parse List.Empty |> List.toArray
```
Comment (knocte) [inline NOnion/Utility/Base32Util.fs:170]: @parhamsaremi this is a false negative for AvoidTooShortNames! Please file a bug about this tomorrow

Comment (parhamsaremi) [reply NOnion/Utility/Base32Util.fs:170]: filed in https://github.com/fsprojects/FSharpLint/issues/592

Comment (knocte) [reply NOnion/Utility/Base32Util.fs:170]: Ok thanks, you can try fixing it (add a test first). There's a chance that it's already fixed (maybe by this commit which is not contained on a publish release yet: https://github.com/fsprojects/FSharpLint/commit/c72dcd61785edc69f6cdbaee20bc40cefbdb1a8e ), but if it is, there's still value in adding the new unit test.

Comment (parhamsaremi) [reply NOnion/Utility/Base32Util.fs:170]: great sure 👍

**Pull request merged**